docs/dev/guidelines/language-guidelines.rst

```diff
@@ -52,6 +52,7 @@ The German language is mainly used for user interaction.
 **Best practices** (written in German for demonstration purposes)
 
 * **Nutze neutrale Formulierungen**: Wenn möglich ist es empfehlenswert, neutrale Formulierungen zu finden. Neben passiven Formen, wie z.B. Studierende, können auch aktive mit „Mensch“ bzw. „Person“ etc. verwendet werden. Das spart „der/die“ und inkludiert alle Menschen unabhängig davon, welche Eigenschaften/Kategorien diese Personen sonst noch mitbringen.
+* **Verwende die Anrede „Du“**:Um eine persönliche und weniger formelle Atmosphäre zu schaffen, verwende „Du“ statt „Sie".
```

Comment:
```suggestion
* **Verwende die Anrede „du“**:Um eine persönliche und weniger formelle Atmosphäre zu schaffen, verwende „du“ statt „Sie".
```
Existing translations use lowercase ‘du’, ‘dich’, …

Please adapt the updated translations in this PR to follow this schema as well.